lib/src/commit_builder.rs

```diff
@@ -119,6 +119,14 @@ impl CommitBuilder<'_> {
         self
     }
 
+    pub fn apply_if<F: FnOnce(Self) -> Self>(self, condition: bool, f: F) -> Self {
```

Comment:
I don't think it's worth modifying the CommitBuilder class for this.

We could do it the boring and somewhat ugly way without an `apply` function. Alternatively, we could get some inspiration from https://docs.rs/apply/latest/src/apply/lib.rs.html#16-31. I don't know it's better to depend on that crate or to include a similar function inside our library.

Reply:
Done.